aten/src/ATen/core/PhiloxRNGEngine.h

```diff
@@ -21,7 +21,7 @@
 namespace at {
 
 // typedefs for holding vector data
-namespace {
+namespace detail {
 
 typedef at::detail::Array<uint32_t, 4> UINT4;
```

Comment:
It looks like these are only used in `philox_engine` so maybe they should just be placed in `class philox_engine`, but this is fine too

Reply:
You're right, but I am a lazy man :)